src/apis/stats_nba_com.py

```diff
@@ -0,0 +1,72 @@
+import json
+import selenium
+import sys
+from selenium import webdriver
+from time import sleep
+
+
+BROWSER_LOAD_DELAY = 6
+
+
+conn = None
+def get_connection():
+    """Returns a StatsNbaComApi connection
+    """
+    global conn
+    if conn is None:
+        conn = StatsNbaComApi()
+    return conn
+
+
+class StatsNbaComApi:
```

Comment:
This class is the API.  The main method it offers is `get`, which  just fetches that URL